charts/tekton-pipelines/templates/buildpacks/pipelines/buildpack.yaml

```diff
@@ -1,5 +1,5 @@
 {{- range $pipeline := .Values.buildpacks.generate }}
-{{$data := dict "pipeline" $pipeline "namespace" $.Release.Namespace }}
+{{$data := dict "pipeline" $pipeline "namespace" $.Release.Namespace "buildOnly" false}}
```

Comment:
I think `1buildOnly` should be taken from some `tekton-apps` variable (for example from TriggerBinding values). So here you will use just variable from TB. So you won't need thse separate buildpack files